vowels.js

```diff
@@ -0,0 +1,19 @@
+var vowels =["a","e","i","o","u"];
+
+var newArray="";
+function countVowels(str){
+
+    var vcount=0;
```

Comment:
I can see you are counting the number of vowels which shouldn't be the case. you need to count the number of duplicates. You variable names should always be explicit enough so that your code can be more readable. eg `vowelCount` is more explicit that `vCount`